src/Compilers/CSharp/Portable/Binder/ForLoopBinder.cs

```diff
@@ -90,12 +90,12 @@ private BoundForStatement BindForParts(ForStatementSyntax node, Binder originalB
                     if (increment.Kind == BoundKind.StatementList)
                     {
                         increment = new BoundBlock(scopeDesignator, locals, ((BoundStatementList)increment).Statements)
-                                            { WasCompilerGenerated = true };
+                        { WasCompilerGenerated = true };
                     }
                     else
                     {
                         increment = new BoundBlock(increment.Syntax, locals, ImmutableArray.Create(increment))
-                                            { WasCompilerGenerated = true };
+                        { WasCompilerGenerated = true };
```

Comment:
another case.   i feel lke indentation should be allowed at certain depths.  either a depth of '1', or 'matching with 'new' should be allowed.

Reply:
➡️ Duplicate of https://github.com/dotnet/roslyn/pull/31142#discussion_r233238801